lib/Runtime/Library/SharedArrayBuffer.h

```diff
@@ -194,7 +194,7 @@ namespace Js
 
         Waiters * m_waiters;
 
-        // Below CS is used for synchronizig access in wait/wake API
+        // Below CS is used for synchronizing access in wait/wake API
```

Comment:
Prefast finds typos now?

Reply:
That being said, there are tools that spell-check comments, but I doubt that we'd find much benefit from using them.

Reply:
I was just taking the opportunity to make a joke. We have typos in comments all over but fixing them like this - as a one-off thing - is probably best.

Reply:
Ah, sorry. 👍

Reply:
np 👌

Reply:
We (I) actually did use some spell checker tools before we open-sourced ChakraCore -- didn't catch everything and more typos have creeped in since then. I support drive-by clean up. :)